sources/api/corndog/Cargo.toml

```diff
@@ -19,6 +19,7 @@ serde_json = "1"
 simplelog = "0.12"
 snafu = "0.7"
 tokio = { version = "~1.32", default-features = false, features = ["macros", "rt-multi-thread"] }  # LTS
+toml = "0.8"
```

Comment:
Can we remove the tokio dependency from these handlers or do we still need them to be async?

Reply:
This is a good point. Without using the HTTP stack, there is nothing left that needs to be async. Please try removing the dependency on `tokio` and remove the async and await keywords everywhere they appear in the program.

Also remove the apiclient dependency, the http dependency, and the models dependency (I hope).

Try `cargo-udeps` or `cargo-machette` for finding unused dependencies.